### DISCUSSION TITLE
Was Occupancy Grid example removed from the new wrapper realese ?

### DISCUSSION THREAD

**Discussion by nadaaqeel** [realsenseai/realsense-ros#2399] [Q&A]
Hello all, 

I am following this tutorial  https://dev.intelrealsense.com/docs/2d-occupancy-map which redirects to the link of the ROS wrapper here
https://github.com/IntelRealSense/realsense-ros/tree/occupancy-mapping, but when I try to clone the repository, it doesn't have the folder of occupancy (and it has some other camera2_description ).
Can you tell me where to find the Occupancy example!

**Comment by sanghun-biel** [comment]
I don't know why, but the official repository seems to have been deleted.
However, there are other stores that have been fork, so I think I should refer to them.
https://github.com/rjwb1/realsense-ros/tree/occupancy-mapping

**Reply by SamerKhshiboun** [reply]
Branch reverted:
https://github.com/IntelRealSense/realsense-ros/tree/old/occupancy-mapping
